### ISSUE TITLE
tagging in register causes crash in ubuntu 14.04, differences in GLUT

### ISSUE THREAD

**Issue by vfonov** [BIC-MNI/Register#1]
Opening two volumes and placing four tags causes register to crash with following message:

XIO:  fatal IO error 11 (Resource temporarily unavailable) on X server ":0"
      after 1706 requests (1706 known processed) with 0 events remaining.

However, copying /usr/lib/x86_64-linux-gnu/libglut.so.3.9.0 from ubuntu 12.04 and using it instead of system one fixes this problem.

**Comment by psteadman** [comment]
I have had this same issue. I tried your solution and it worked. I just wanted to add another point, just incase other users have troubles. The symbolic-link to usr/lib/x86_64-linux-gnu/libglut.so.3 doesn't always point to the correct usr/lib/x86_64-linux-gnu/libglut.so.3.9.0. This symbolic-link sometimes points to the original usr/lib/x86_64-linux-gnu/libglut.so.3.9.0.orig - not sure if a package I installed later made this change or what. 

So just make sure both symbolic-links for libglut.so point to the right library. That is usr/lib/x86_64-linux-gnu/libglut.so and usr/lib/x86_64-linux-gnu/libglut.so.3 point to usr/lib/x86_64-linux-gnu/libglut.so.3.9.0 that was copied from 12.04
